src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksUtilityServiceImpl.java

```diff
@@ -164,6 +167,32 @@ public String trimFieldToMax(String field, String fieldName) {
 	public String convertPhoneNumber(String phoneNumber) {
 		return getPhoneNumberService().formatNumberIfPossible(phoneNumber);
 	}
+	
+	@Override
+	public boolean shouldVendorBeSentToPaymentWorks(VendorDetail vendorDetail) {
+		boolean sendToPaymentWorks = vendorDetail.isActiveIndicator()
+				&& !(StringUtils.equals(vendorDetail.getVendorHeader().getVendorOwnershipCategoryCode(), "US")
```

Comment:
Should we have a system parameter with a list of vendor ownership category codes that are not supposed to be sent to Payment Works? Then this code could then call a ParameterEvaluator to check if the vendor ownership category code is allowed.